qa/restart-upgrade/src/test/resources/processor/IndexMappings.json

```diff
@@ -0,0 +1,32 @@
+{
+  "settings": {
+    "index": {
+      "knn": true,
+      "knn.algo_param.ef_search": 100,
+      "refresh_interval": "30s",
+      "default_pipeline": "%s"
+    },
+    "number_of_shards": 6,
+    "number_of_replicas": 0
```

Comment:
should we go with 3 primaries and 1 replica?

Reply:
https://github.com/opensearch-project/neural-search/pull/515#discussion_r1437284537

Reply:
I am getting below error for 3 / 1 configration of shards
```
org.opensearch.neuralsearch.bwc.SemanticSearchIT > testSemanticSearch_E2EFlow FAILED
    org.opensearch.client.ResponseException: method [PUT], host [http://127.0.0.1:40221/], URI [/neuralsearch-bwc-testsemanticsearch_e2eflow/_doc/2?refresh=true], status line [HTTP/1.1 400 Bad Request]
    {"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"Model not ready yet. Please run this first: POST /_plugins/_ml/models/eJaKyYwBOutJBPQEmqxh/_deploy"}],"type":"illegal_argument_exception","reason":"Model not ready yet. Please run this first: POST /_plugins/_ml/models/eJaKyYwBOutJBPQEmqxh/_deploy"},"status":400}
        at __randomizedtesting.SeedInfo.seed([7FBAAD9E29825EEB:C9A802068E91C12C]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:385)
        
       ```